hawkbit-dmf-amqp/src/main/java/org/eclipse/hawkbit/amqp/AmqpMessageHandlerService.java

```diff
@@ -207,6 +213,22 @@ private Message handleAuthentifiactionMessage(final Message message) {
         return getMessageConverter().toMessage(authentificationResponse, messageProperties);
     }
 
+    private LocalArtifact findLocalArtifactByFileResource(final FileResource fileResource) {
+        LocalArtifact localArtifact = null;
```

Comment:
I prefer direct return without a local variable. You're doing nothing with the local variable, so you can remove it